documentation/modules/adding-source-provider.adoc

```diff
@@ -6,11 +6,11 @@
 ifdef::vmware[]
 = Adding a VMware vSphere source provider
 
-You can migrate VMware vSphere VMs from VMware vCenter or from a VMWare ESX/ESXi server. In {project-short} versions 2.6 and later, you can migrate directly from an ESX/ESXi server, without going through vCenter, by specifying the SDK endpoint to that of an ESX/ESXi server.
+You can migrate {vmw} vSphere VMs from {vmw} vCenter or from a {vmw} ESX/ESXi server. In {project-short} versions 2.6 and later, you can migrate directly from an ESX/ESXi server, without going through vCenter, by specifying the SDK endpoint to that of an ESX/ESXi server.
```

Comment:
And also by choosing the option of "ESXi" for the "Endpoint type" field within the create provider form.